src/test/resources/application-test.properties

```diff
@@ -7,5 +7,6 @@ connection.maxAttempts=6
 connection.backoff=10000
 cache.prefetch=1
 api.chunksize=10485760
+tree=false

```

Comment:
Do this means that the default value for tree is false?

Reply:
For only test cases defaults tree is disabled but I have written test cases for both enable and disable